packages/core/useCssVar/index.ts

```diff
@@ -55,12 +55,11 @@ export function useCssVar(
         old[0].style.removeProperty(old[1])
       updateCssVar()
     },
-    { immediate: true },
   )
 
   watch(
-    variable,
-    (val) => {
+    [variable, elRef],
+    ([val]) => {
```

Comment:
Maybe not such a big difference, but I think we should add the new element here as well?
```suggestion
    [variable, elRef],
    ([val, el]) => {
```

I can't suggest the full function because it's not considered a change. So the function should use el too. 😄 what do you think?

Reply:
Sure, no problem, I think it's great.